doc/reference/plutus/reference-script-example.md

```diff
@@ -0,0 +1,146 @@
+# Plutus Scripts
+
+## What is a reference script?
+
+A reference script is a script that exists at a particular transaction output. It can be used to witness, for example, a UTxO at the corresponding script address of said reference script. Why is this useful? It means we no longer have to include the script in the transaction, greatly reducing its size
```

Comment:
Suggest changing to:
'This is useful because the script does not have to be included in the transaction anymore, which significantly reduces the transaction size.